python/lsst/meas/extensions/gaap/_gaussianizePsf.py

```diff
@@ -118,13 +118,19 @@ def run(self, exposure: lsst.afw.image.Exposure, center: lsst.geom.Point2D,  # n
         result = self._buildCellSet(exposure, center, targetPsfModel)
         kernelCellSet = result.kernelCellSet
         targetPsfModel = result.targetPsfModel
-        fwhmScience = exposure.getPsf().computeShape().getDeterminantRadius()*sigma2fwhm
+        fwhmScience = exposure.getPsf().computeShape(center).getDeterminantRadius()*sigma2fwhm
         fwhmModel = targetPsfModel.computeShape().getDeterminantRadius()*sigma2fwhm
+        self.log.debug(f"Ratio of model to science PSF = {fwhmModel/fwhmScience}")
```

Comment:
I can't keep up with the proper use of format strings in logging, but I will note that the use of `f` here does not match the other `self.log.debug` statements I see in this file.

Reply:
For logging, we're not supposed to use `f` strings.  It should be `self.log.debug("Ratio of model to science PSF = %f", fwhmModel/fwhmScience)`

Reply:
I am also happy to remove this logging from here. It will appear something like: ``psfMatch DEBUG: Ratio of model to science PSF = 1.15``, which may still not be super helpful.

Reply:
Totally your call given that it's at `debug` level 😄